docs/config.md

```diff
@@ -1,174 +1,39 @@
-# Configuration
+# OVOS Configuration
+When you first start OVOS, there should not be any configuration needed to have a working device.
 
-- [Reading Configuration](#reading-configuration)
-- [Configuring Configuration](#configuring-configuration)
-    * [protected_keys](#protected-keys)
-    * [disable_user_config](#disable-user-config)
-    * [disable_remote_config](#disable-remote-config)
-- [Meta Configuration](#meta-configuration)
-    * [ovos.conf](#ovosconf)
+**NOTE** To continue with the examples, you will need access to a shell on your device.  This can be achevied with SSH.  Connect to your device with the command `ssh ovos@<device_ip_address>` and enter the password `ovos`.
```

Comment:
```suggestion
**NOTE** To continue with the examples, you will need access to a shell on your device.  This can be achevied with SSH.  Connect to your device with the command `ssh ovos@<device_ip_address>` and enter the password `ovos`.
```
```suggestion
**NOTE** To continue with the examples, you will need access to a shell on your device.  This can be achieved with SSH.  Connect to your device with the command `ssh ovos@<device_ip_address>` and enter the password `ovos`.
```

Reply:
```suggestion
**NOTE** To continue with the examples, you will need access to a shell on your device. This can be achieved with SSH. Connect to your device with the command `ssh ovos@<device_ip_address>` and enter the password `ovos`.
```

That double space is erroneous too, right?

Reply:
I have always double spaced after a `.`  Thats how I was taught in highschool 30 years ago.  So ya, there are several of them